cmip6_downscaling/methods/common/tasks.py

```diff
@@ -545,7 +552,10 @@ def pyramid(
     dta = _pyramid_postprocess(dta, levels=levels, other_chunks=other_chunks)
 
     # write to target
-    dta.to_zarr(target, mode='w')
+
+    blocking_to_zarr(ds=dta, target=target, validate=True, write_empty_chunks=True)
+    blocking_to_zarr(ds=dta, target=target, validate=True, write_empty_chunks=True)
+
```

Comment:
@norlandrhagen, if you haven't done so already, we should check if this works on datatree objects. My trivial tests  only covered xarray datasets with a single zarr group.